examples/sandbox_resources.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from nkms.characters import Ursula
+from nkms.network.node import NetworkyStuff
+from nkms.crypto.powers import SigningPower, EncryptingPower
+
+
+class SandboxNetworkyStuff(NetworkyStuff):
```

Comment:
I thought you said you changed the name to `NetworkMiddleware`?

Did this get overwritten or something?

Reply:
No, I haven't changed the class yet; just composed it on each Character as `network_middleware` - we can do the full refactor if we all like it, which it sounds like we do.